modin/core/io/text/csv_dispatcher.py

```diff
@@ -21,4 +21,20 @@
 class CSVDispatcher(TextFileDispatcher):
     """Class handles utils for reading `.csv` files."""
 
-    read_callback = pandas.read_csv
+    def read_callback(*args, **kwargs):
```

Comment:
We need to provide correct docstring for `read_callback` method since this method displayed in the docs.